README.md

```diff
@@ -511,6 +511,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Dropwizard](https://dropwizard.github.io/dropwizard/) - Opinionated framework for setting up modern web applications with Jetty, Jackson, Jersey and Metrics.
 * [Feign](https://github.com/Netflix/feign) - HTTP client binder inspired by Retrofit, JAXRS-2.0, and WebSocket.
 * [Jersey](https://jersey.java.net/) - JAX-RS reference implementation.
+* [Rapidoid](http://www.rapidoid.org/) - Open source (Apache license). Simple. Secure. Extremely fast!
```

Comment:
Uninformative description, #283 had a good one but closed his PR for reasons unknown:

`A Simple, secure and extremely fast framework consisting of embedded HTTP server, GUI components and dependency injection.`